### PR TITLE
[FIX] cetmix_tower_server:t3960 Fix replacing of variables after create server from template

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Comment by CetmixGitDrone** [comment]
Congratulations, PR rebased to [14.0-dev](https://github.com/cetmix/cetmix-tower/commits/14.0-dev).

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]
Consider adding the following test scenario:

- Variable A is defined in the  Server Template == "pizza"
- Variable B is defined in the  Server Template == "green"


New server is created from the template with the following variable values:

- Variable B  == "green"
- Variable C == "dog"

Expected result is: new server is created with the following variable values:

- Variable A == "pizza"
- Variable B  == "green"
- Variable C == "dog"

**Comment by GabbasovDinar** [comment]
> Consider adding the following test scenario:
> 
> * Variable A is defined in the  Server Template == "pizza"
> * Variable B is defined in the  Server Template == "green"
> 
> New server is created from the template with the following variable values:
> 
> * Variable B  == "green"
> * Variable C == "dog"
> 
> Expected result is: new server is created with the following variable values:
> 
> * Variable A == "pizza"
> * Variable B  == "green"
> * Variable C == "dog"

Fixed. 

P.S. I used exists variables for such case

File: cetmix_tower_server/tests/test_server_template.py [hunk 1]
```diff
@@ -151,12 +159,43 @@ def test_create_server_from_template_action(self):
             ssh_username="test",
             ssh_password="test",
             configuration_variables={
+                self.variable_version.name: "test server version",
```
Comment (ivs-cetmix) [inline cetmix_tower_server/tests/test_server_template.py]: We use `reference` field here not `name`

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]

**Review by ivs-cetmix — APPROVED** [review]
Code review LGTM

**Comment by CetmixGitDrone** [comment]
This PR has the `approved` label and has been created more than 5 days ago. It should therefore be ready to merge by a maintainer (or a PSC member if the concerned addon has no declared maintainer). 🤖

**Comment by CetmixGitDrone** [comment]
This PR looks fantastic, let's merge it!
Prepared branch [14.0-dev-ocabot-merge-pr-100-by-ivs-cetmix-bump-nobump](https://github.com/cetmix/cetmix-tower/commits/14.0-dev-ocabot-merge-pr-100-by-ivs-cetmix-bump-nobump), awaiting test results.

**Comment by CetmixGitDrone** [comment]
Congratulations, your PR was merged at 50c240f9c4b628043fddaf76ee5e4d84c30866a1. Thanks a lot for contributing to cetmix. ❤️

**Pull request merged**